### PR TITLE
Doing the Minimum

### PR DESCRIPTION
**Pull request by mjmagee991**
Implemented basic cooling logic

### CODE REVIEW

File: targets/TMS/main.cpp [hunk 1]
```diff
@@ -117,7 +119,13 @@ int main() {
     log::LOGGER.log(log::Logger::LogLevel::DEBUG, "Logger initialized.");
     timer.stopTimer();
 
+    // Set up TMS and necessary device drivers
     TMS::TMS tms;
+    IO::PWM& pwm = IO::getPWM<IO::Pin::PC_0>();
+    TMS::HeatPump pump = TMS::HeatPump(pwm);
+    TMS::RadiatorFan fans[] = {
+        TMS::RadiatorFan(IO::getGPIO<IO::Pin::PC_1>()),
+        TMS::RadiatorFan(IO::getGPIO<IO::Pin::PB_0>())};
```
Comment (AarC10) [inline targets/TMS/main.cpp:128]: I believe the curly brace should be on a new line

**Review by AarC10 — APPROVED** [review]
Looks good to me besides what I believe to be a minor styling issue. Good job.

**Review by cbolles — APPROVED** [review]
Looks good!

File: targets/TMS/main.cpp (hunk 1 above)
Comment (cbolles) [reply targets/TMS/main.cpp:128]: I think that is correct, the code formatter would have caught that if it didn't match out style

**Review by zsteig — APPROVED** [review]

**Pull request merged**